react/spring-page/src/components/Main/Main.js

```diff
@@ -0,0 +1,61 @@
+import React, { useState, useEffect } from 'react';
+import './Main.css';
+import { useDispatch, useSelector } from 'react-redux';
+import { arrDescription } from '../../constants/SpringInfo';
+import { Description } from '../Description/Description';
+import { setArrForSearch } from '../../redux/actions/action';
+
+export function Main() {
+  const [name, setName] = useState('');
+  const dispatch = useDispatch();
+  const arrData = useSelector((state) => state.searchTags.arr);
```

Comment:
I told you that several times in DM - this is a hideous name. `arr`? What? Array? There are tons of arrays out there.